### ISSUE TITLE
Mark's viability analysis: Questions and comments about Figure 1D

### ISSUE THREAD

**Issue by rolandknapp** [SNARL1/translocation#12]
- Assuming that Year 0 corresponds to 2006 (year of first translocation), then Year 16 is 2022. No data are available yet for 2022, so what is being plotted for the observed trajectory?
- In determining whether model could reproduce observed trajectory at 70550, chose 100 best fit trajectories. Justification? This will produce an overly optimistic assessment. Why not choose trajectories at random?
- The difference between modeled and actual frog abundance is explained quite well by average snow-water-equivalent (SWE) from t-2 to t0 (i.e., rolling average of SWE in previous 3 winters). Recruitment (= higher abundance in figure) much more likely when ave_SWE is < 0.6. Shows even better when 2022 frog abundance is included (recruitment estimate likely to show 200+ new adults). Sent SWE time series to Mark on 28-Sep-2022 (attached here). 
[DanaMeadows_Snowpack.xlsx](https://github.com/SNARL1/translocation/files/9779106/DanaMeadows_Snowpack.xlsx)

**Comment by mqwilber** [comment]
- In the previous version of Fig. 1D, Year 1 corresponded to 2006 (i.e., the year of translocation).  So the 16 "years" were really 16 time points where there was an abundance estimate.  I have updated the x-axis ticks in Fig. 1D to be labeled as years.
- I have added a justification and citation for the approach of selecting the 100 best fit trajectories based on Approximate Bayesian Computing [here](https://github.com/SNARL1/translocation/blob/main/doc/manuscript/translocation.qmd).
